zkevm-circuits/src/mpt_circuit/account_leaf.rs

```diff
@@ -430,18 +465,39 @@ impl<F: Field> AccountLeafConfig<F> {
             let address = lo + hi;
 
             ifx! {not!(config.parent_data[false.idx()].is_placeholder) => {
-                ctx.mpt_table.constrain(
-                    meta,
-                    &mut cb.base,
-                    address.clone(),
-                    proof_type.clone(),
-                    Word::<Expression<F>>::new([0.expr(), 0.expr()]),
-                    config.main_data.new_root.expr(),
-                    config.main_data.old_root.expr(),
-                    Word::<Expression<F>>::new([new_value_lo, new_value_hi]),
-                    Word::<Expression<F>>::new([old_value_lo.clone(), old_value_hi.clone()]),
-                );
+                ifx! {not!(config.is_non_existing_account_proof) => {
+                    ctx.mpt_table.constrain(
+                        meta,
+                        &mut cb.base,
+                        address.clone(),
+                        proof_type.clone(),
+                        Word::<Expression<F>>::new([0.expr(), 0.expr()]),
```

Comment:
We can now use `Word::zero()` here now
```suggestion
                        Word::zero(),
```